src/pages/task/Metadata.tsx

```diff
@@ -183,6 +186,12 @@ export const Metadata: React.FC<Props> = ({ loading, data, error, taskId }) => {
           ))}
         </span>
       ) : null}
+      {oomTracker && oomTracker.detected && (
+        <P2>
+          Out of Memory Kill detected
+          {oomTracker.pids ? `(PIDs: ${oomTracker.pids.join(", ")}` : ""} )
```

Comment:
`oomTracker.pids` should actually be `oomTracker?.pids` since `oomTracker` could be undefined based on the assignment in line 54